config/strides/synapse-login-strides-params.yaml

```diff
@@ -0,0 +1,41 @@
+template_path: remote/ssm-parameters.j2
+stack_name: synapse-login-strides-params
+hooks:
+  before_launch:
+    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/master/ssm-parameters.j2 --create-dirs -o templates/remote/ssm-parameters.j2"
+stack_tags:
+  Department: "Platform"
+  Project: "Infrastructure"
+  OwnerEmail: "it@sagebase.org"
+sceptre_user_data:
+  Prefix: /service-catalog/
+  Parameters  :
+    - Name: SynapseOauthClientId
+      Value: 100053
```

Comment:
the client id will change

Reply:
The client ID is 100063

Reply:
where is this OIDC client secret @brucehoff ?

Reply:
The OIDC client ID and secret are in LastPass, as mentioned in Jira: https://sagebionetworks.jira.com/browse/SC-235?focusedCommentId=121376&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-121376